src/server/server.js

```diff
@@ -72,7 +72,7 @@ app.get('/journalism/teams/:teamname', (req, res) => {
   journalismTeam(res, req.params.teamname);
 });
 
-app.get('/journalism/player/:playername', (req, res) => {
+app.get('/journalism/players/:playername', (req, res) => {
```

Comment:
Rather than changing this to /players, I think we should actually change the other to /team - being singular seems to make more sense to me.

I'm cool with either though